packages/gitbook/src/components/Search/server-actions.tsx

```diff
@@ -258,7 +260,9 @@ async function searchSiteContent(
     return (
         await Promise.all(
             searchResults.map(async (spaceItem) => {
+                const sections = getSiteStructureSections(structure, { ignoreGroups: true });
                 const siteSpace = findSiteSpaceById(structure, spaceItem.id);
+                const siteSection = sections.find((section) => section.id === siteSpace?.section);
```

Comment:
Do we want to take into consideration the section group as well in the breadcrumb?